src/styles/getModalStyles.ts

```diff
@@ -1,25 +1,60 @@
-import CONST from '../../CONST';
-import variables from '../variables';
-import themeColors from '../themes/default';
-import styles from '../styles';
+import {CSSProperties} from 'react';
+import {ViewStyle} from 'react-native';
+import {ValueOf} from 'type-fest';
+import {ModalProps} from 'react-native-modal';
+import CONST from '../CONST';
+import variables from './variables';
+import themeColors from './themes/default';
+import styles from './styles';
 
-const getCenteredModalStyles = (windowWidth, isSmallScreenWidth, isFullScreenWhenSmall = false) => ({
-    borderWidth: styles.centeredModalStyles(isSmallScreenWidth, isFullScreenWhenSmall).borderWidth,
-    width: isSmallScreenWidth ? '100%' : windowWidth - styles.centeredModalStyles(isSmallScreenWidth, isFullScreenWhenSmall).marginHorizontal * 2,
-});
+function getCenteredModalStyles(windowWidth: number, isSmallScreenWidth: boolean, isFullScreenWhenSmall = false): ViewStyle {
+    return {
+        borderWidth: styles.centeredModalStyles(isSmallScreenWidth, isFullScreenWhenSmall).borderWidth,
+        width: isSmallScreenWidth ? '100%' : windowWidth - styles.centeredModalStyles(isSmallScreenWidth, isFullScreenWhenSmall).marginHorizontal * 2,
+    };
+}
+
+type ModalType = ValueOf<typeof CONST.MODAL.MODAL_TYPE>;
```

Comment:
```suggestion
type Modal = ValueOf<typeof CONST.MODAL.MODAL_TYPE>;
```

From guidelines:

> Do not postfix type aliases with `Type`

Reply:
I think this is different, because it is the type of a modal (like there are couple variants/types of modal). So I used word 'Type' in a different context here and I think it's okay as it is. Wdyt? @fabioh8010

Reply:
Ok, we can make this exception, but let's be careful as it can be pretty ambiguous sometimes 😅